.eslintrc.js

```diff
@@ -26,6 +31,23 @@ module.exports = {
     'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
     'quotes': ['error', 'single'],
     'semi': ['error', 'always'],
+    'jsdoc/check-tag-names': [
+      'error',
+      {
+        'definedTags': [
+          'model',
+          'slot'
```

Comment:
you were using `@model` and `@slot` in a few places. this defines them here to prevent lint warnings.

Reply:
Yeah, those are actually used in Vue Styleguidist. I had it working on an old versions, but couldnt get it to work right with Vuepress. I'll come back to it because it's really nice when your docs write themselves. For now these dont actually do anything until I can get that hooked up again.